### PR TITLE
add PyMieScatt

### PR DESCRIPTION
**Pull request by kmuehlbauer**

### CODE REVIEW

File: recipes/pymiescatt/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (hmaarrfk) [inline recipes/pymiescatt/LICENSE.md]: FYI: https://github.com/bsumlin/PyMieScatt/pull/7

File: recipes/pymiescatt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - scipy
+    - matplotlib
+    - shapely
+
+test:
+  imports:
+    - PyMieScatt
+
+about:
+  home: http://air.eece.wustl.edu/people/ben-sumlin/
```
[the remark this answers is not in the document]
Comment (hmaarrfk) [reply recipes/pymiescatt/meta.yaml]: https://github.com/bsumlin/PyMieScatt

**Comment by hmaarrfk** [comment]
It seem the package has 2 license files. One GPL, one MIT

Maybe ask upstream to clean that up first: (I've just done that, but I probably won't follow up: https://github.com/bsumlin/PyMieScatt/pull/7)

https://github.com/bsumlin/PyMieScatt/blob/master/PyMieScatt/LICENSE
https://github.com/bsumlin/PyMieScatt/blob/master/LICENSE.md

Finally, maybe ask upstream to use tags to make "releases". You can then use those to get your updated version.

File: recipes/pymiescatt/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (kmuehlbauer) [reply recipes/pymiescatt/meta.yaml]: @hmaarrfk That was the corresponding line from the setup.py. Should we drop this and add the repo-url instead?

**Comment by kmuehlbauer** [comment]
@hmaarrfk Yes, I asked to resolve this GPL/MIT clash in https://github.com/bsumlin/PyMieScatt/issue/1. Also asked about a version update on PyPI here https://github.com/bsumlin/PyMieScatt/issues/6.

Let's see how this develops. Thanks!

File: recipes/pymiescatt/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (hmaarrfk) [reply recipes/pymiescatt/meta.yaml]: Yeah this seems like some of his PhD work and he probably didn’t have the whole infrastructure setup to link correctly. I think the github url is the most stable.

**Comment by hmaarrfk** [comment]
Yeah. We have to wait and see for now. Anyway. I can’t merge this so somebody else will have to review it too.

**Comment by kmuehlbauer** [comment]
@hmaarrfk No worries. :+1:

**Comment by ocefpaf** [comment]
Thanks @hmaarrfk for the review! I guess that the remaining issues here can be fixed later in the feedstock.

**Comment by kmuehlbauer** [comment]
@ocefpaf Thanks for merging. Trying to clear things up with upstream in the feedstock.

**Pull request merged**